src/FSharpLint.Console/Output.fs

```diff
@@ -34,29 +34,30 @@ type StandardOutput () =
         Console.ForegroundColor <- originalColour
 
     interface IOutput with
+
         member _.WriteInfo (info:string) = writeLine info ConsoleColor.White Console.Out
         member this.WriteWarning (warning:Suggestion.LintWarning) =
             let highlightedErrorText = highlightErrorText warning.Details.Range warning.ErrorText
-            let ruleUrlHint = sprintf "See https://fsprojects.github.io/FSharpLint/how-tos/rules/%s.html" warning.RuleIdentifier
-            let str = warning.Details.Message + Environment.NewLine + highlightedErrorText
-                    + Environment.NewLine + ruleUrlHint
+            let ruleUrlHint = $"See https://fsprojects.github.io/FSharpLint/how-tos/rules/%s{warning.RuleIdentifier}.html"
+            let str = $"{warning.Details.Message}{Environment.NewLine}{highlightedErrorText}{Environment.NewLine}{ruleUrlHint}"
             writeLine str ConsoleColor.Yellow Console.Out
             String.replicate 80 "-" |> (this :> IOutput).WriteInfo
         member _.WriteError (error:string) =  writeLine error ConsoleColor.Red Console.Error
 
 type MSBuildOutput () =
+
     interface IOutput with
+
         member _.WriteInfo (info:string) = Console.Out.WriteLine info
         member _.WriteWarning (warning:Suggestion.LintWarning) =
-            sprintf "%s(%d,%d,%d,%d):FSharpLint warning %s: %s"
+            fprintf Console.Out "%s(%d,%d,%d,%d):FSharpLint warning %s: %s"
```

Comment:
@xperiandri Should this be ```fprintfn``` as it used to do Out.WriteLine?

(ref https://github.com/fsprojects/FSharpLint/issues/740#issuecomment-3062105171)